okrs/2019-q1.md

```diff
@@ -0,0 +1,24 @@
+## Objective: ProtoSchool is a growing Top Level Project.
+
+| Key Result | Assignee | Mid-Q Score | Projected Score | Final Score |
+| ---------- | -------- | ----------- | --------------- | ----------- |
+| [ProtoSchool has its own Roadmap.](https://github.com/ProtoSchool/protoschool.github.io/issues/104) | @terichadbourne | | | |
+| [ProtoSchool has rich metrics that can be used to improve it in the future.](https://github.com/ProtoSchool/protoschool.github.io/issues/100) | @terichadbourne | | | |
```

Comment:
I'm comfortable with this if "rich" is defined loosely. We've talked about the Google event tracking... Is there something else you'd want to see achieved to meet this goal?

Reply:
Correct, it just means that we've instrumented some custom events for specific things we want to track.